runtime/compiler/control/CompilationRuntime.hpp

```diff
@@ -817,6 +817,12 @@ class CompilationInfo
    void acquireLogMonitor();
    void releaseLogMonitor();
 
+   // Synchronize transient class caching
+   bool isClassLoaderTransient(void *ClassLoader);
```

Comment:
If you can avoid  `void *`, you should. 
In OpenJ9 coding convention param names don't start with capital letters.
Let's rename this function `isClassLoaderMarkedTransient`